diff --git a/tests/Makefile.am b/tests/Makefile.am
index 3644091..fae8e7a 100644
--- a/tests/Makefile.am
+++ b/tests/Makefile.am
@@ -126,4 +126,14 @@ all-local:
 	chmod a+x $(srcdir)/check-test-fullmode-with-stun.sh
 	chmod a+x $(srcdir)/test-pseudotcp-random.sh
 
+nemotestsbindir = /opt/tests/libnice/bin
+nemotestsbin_PROGRAMS = $(check_PROGRAMS)
+nemotestsbin_SCRIPTS = $(dist_check_SCRIPTS)
+
+libnice-tests.list:
+	$(AM_V_GEN)echo $(check_PROGRAMS) > $@
+
+nemotestsdatadir = /opt/tests/libnice/data
+nobase_nemotestsdata_DATA = libnice-tests.list
+
 EXTRA_DIST = libnice.supp
